pkg/kubelet/volumemanager/populator/desired_state_of_world_populator.go

```diff
@@ -419,15 +441,15 @@ func (dswp *desiredStateOfWorldPopulator) getPVCExtractPV(
 
 	if pvc.Status.Phase != v1.ClaimBound || pvc.Spec.VolumeName == "" {
 
-		return "", "", fmt.Errorf(
+		return "", "", "", fmt.Errorf(
 			"PVC %s/%s has non-bound phase (%q) or empty pvc.Spec.VolumeName (%q)",
 			namespace,
 			claimName,
 			pvc.Status.Phase,
 			pvc.Spec.VolumeName)
 	}
 
-	return pvc.Spec.VolumeName, pvc.UID, nil
+	return pvc.Spec.VolumeName, pvc.UID, *pvc.Spec.VolumeMode, nil
```

Comment:
Can VolumeMode be nil?  I think @screeley44 is writing a helper function to get the volume mode that will default it.

Reply:
yes. I also remember that I have similar method, but I'll use @screeley44 's one.